### ISSUE TITLE
Including PAR measurements

### ISSUE THREAD

**Issue by wwieder** [NEONScience/NCAR-NEON#12]
Let's include PAR `DP1.00024.001` data too!  This would be a useful  diagnostic to plot against GPP to  evaluate ACI curves at sites and in the model.

**Comment by wwieder** [comment]
Can we also include direct and diffuse measurements `DP1.00014.001`, also for model diagnostics.

**Comment by ddurden** [comment]
Yes, this should be a pretty straightforward addition. We should discuss in our meeting this afternoon.

**Comment by wwieder** [comment]
Not high priority, but has this been done?
